security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -12,153 +12,28 @@ These release notes track the development of {cert-manager-operator}.
-
-The following advisory is available for the {cert-manager-operator} 1.11.1:
-
-* link:https://access.redhat.com/errata/RHEA-2023:3439[RHEA-2023:3439]
-
-Version `1.11.1` of the {cert-manager-operator} is based on the upstream cert-manager version `v1.11.1`. For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.11/#v1111[cert-manager project release notes for v1.11.1].
-
-[id="cert-manager-operator-1.11.1-new-features-and-enhancements"]
+[id="cert-manager-operator-new-features-1.13"]
 === New features and enhancements
 
-This is the general availability (GA) release of the {cert-manager-operator}.
-
-[id="cert-manager-log-level-1.11.1"]
-==== Setting log levels for cert-manager and the {cert-manager-operator}
-* To troubleshoot issues with cert-manager and the {cert-manager-operator}, you can now configure the log level verbosity by setting a log level for cert-manager and the {cert-manager-operator}. For more information, see xref:../../security/cert_manager_operator/cert-manager-log-levels.adoc#cert-manager-log-levels[Configuring log levels for cert-manager and the {cert-manager-operator}].
-
-[id="cert-manager-authentication-aws-1.11.1"]
-==== Authenticating the {cert-manager-operator} with AWS
-* You can now configure cloud credentials for the {cert-manager-operator} on AWS clusters with Security Token Service (STS) and without STS. For more information, see xref:../../security/cert_manager_operator/cert-manager-authenticate-aws.adoc#cert-manager-authenticate-aws[Authenticating the {cert-manager-operator} on AWS  Security Token Service] and xref:../../security/cert_manager_operator/cert-manager-authentication-non-sts.adoc#cert-manager-authentication-non-sts[Authenticating the {cert-manager-operator} on AWS].
-
-[id="cert-manager-authentication-gcp-1.11.1"]
-==== Authenticating the {cert-manager-operator} with GCP
-* You can now configure cloud credentials for the {cert-manager-operator} on GCP clusters with Workload Identity and without Workload Identity. For more information, see xref:../../security/cert_manager_operator/cert-manager-authenticate-gcp.adoc#cert-manager-authenticate-gcp[Authenticating the {cert-manager-operator} with GCP Workload Identity] and xref:../../security/cert_manager_operator/cert-manager-authenticate-non-sts-gcp.adoc#cert-manager-authenticate-non-sts-gcp[Authenticating the {cert-manager-operator} with GCP]
+* You can now manage certificates for xref:../../security/certificates/api-server.adoc#api-server-certificates[API Servers] and xref:../../security/certificates/replacing-default-ingress-certificate.adoc#replacing-default-ingress[Ingress] by using the {cert-manager-operator}.
 
-[id="cert-manager-operator-1.11.1-bug-fixes"]
-=== Bug fixes
+* With the latest release, the scope of {cert-manager-operator}, which was previously limited to the AMD64 platform, has now been expanded to include support for managing certificates on IBM Z/Power and ARM64 platforms.
```

Comment:
I think RH docs prefer, `amd64` / `x86_64` (I was referring to https://docs.openshift.com/container-platform/4.13/cli_reference/tkn_cli/installing-tkn.html#installing-tkn-on-linux, but please double check if this is correct).

Also, would be good to add the short names for the other platforms included maybe:
- `arm64`
- `ppc64le`
- `s390x`

Reply:
Since the context here to name the platform and not a code, per our [guidelines](https://redhat-documentation.github.io/supplementary-style-guide/#AMD64), it has to be uppercase.